### ISSUE TITLE
Does not work with rollup.

### ISSUE THREAD

**Issue by Dharmadurai95** [wojtekmaj/react-pdf#439]
This library does not work with rollup. 
The error is :
```
[!] Error: 'Document' is not exported by node_modules/react-pdf/dist/entry.js
https://github.com/rollup/rollup/wiki/Troubleshooting#name-is-not-exported-by-module
src/elements/scanner/scannerDrawer/index.tsx (5:8)
5: import {Document,Page} from "react-pdf";
           ```

**Comment by wojtekmaj** [comment]
Likely caused by us using CommonJS modules by default; will be fixed by #423.

**Comment by creativetim** [comment]
This is currently an issue for an implementation at my company as well. Would love to see https://github.com/wojtekmaj/react-pdf/issues/423 incorporated.
